web/assignments.go

```diff
@@ -139,19 +142,57 @@ func (s *AutograderService) createReview(query *pb.Review) (*pb.Review, error) {
 		return nil, fmt.Errorf("Failed to create a new review for submission %d to assignment %s: all %d reviews already created",
 			submission.ID, assignment.Name, assignment.Reviewers)
 	}
-	query.Edited = time.Now().Format("02 Jan 15:04")
+	query.Edited = time.Now().Format(reviewLayout)
```

Comment:
Suggest renaming `query` to `review`, since it now holds review data as well. Same applies to next method as well.

Reply:
Done.